weave-js/src/core/model/graph/editing/hash.ts

```diff
@@ -3,7 +3,7 @@ import _ from 'lodash';
 import type {EditingNode, EditingOp} from './types';
 
 // TODO: is this unique enough for now?
-const cyrb53 = (str: string, seed = 0) => {
+export const cyrb53 = (str: string, seed = 0) => {
```

Comment:
lol this is crazy

Reply:
perfect stack overflow copypasta https://stackoverflow.com/questions/7616461/generate-a-hash-from-string-in-javascript